app/assets/stylesheets/alchemy/selects.scss

```diff
@@ -17,7 +17,7 @@ select {
       @include button($base-color: $white, $padding: 0 5px);
       display: block;
       font-weight: normal;
-      border-color: $button-border-color;
```

Comment:
Cool, no problem. I can switch back super easily. Were there any specific button variables you had in mind to override the Bourbon defaults? I think everything looks preeeetty close right now, so I want to make sure I'm getting exactly what you're looking for so we can knock this out and get these things in 3.5 as you requested.